conftest.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import cupy as cp
+import numpy as np
+import pytest
+from holoscan.core import Application, Fragment
+
+
+@pytest.fixture
+def app():
+    return Application()
+
+
+@pytest.fixture
+def fragment():
+    return Fragment()
+
+
+@pytest.fixture
+def config_file():
+    yaml_file_dir = os.path.dirname(__file__)
+    config_file = os.path.join(yaml_file_dir, "operator_parameters.yaml")
+    return config_file
```

Comment:
It doesn't look like this fixture is currently being used, but I agree we should provide it. I know we had a few of these in Holoscan SDK where this one with `operator_parameters.yaml` had many parameters across various operators. In hindsight, a better approach may be to use a factory style so we can specify a specific file for a given operator:

e.g. something like
```py
@pytest.fixture
def config_file_factory():
    yaml_file_dir = os.path.dirname(__file__)
    def _factory(filename):
        return os.path.join(yaml_file_dir, filename)
    return _factory
```

example usage:
```py
def test_with_config(config_file_factory):
    config_file = config_file_factory("my_operator.yaml")
```

Reply:
Thanks, that makes sense! I also set default values for both filename and directory, so it can be used without any arguments. How does it look to you?

Reply:
Yes, thanks for updating it